common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -230,15 +233,30 @@ define('WordCloudMain', [], function () {
         $.each(response.student_words, function (word, stat) {
             var percent = (response.display_student_percents) ? ' ' + (Math.round(100 * (stat / response.total_count))) + '%' : '';
 
-            studentWordsKeys.push('<strong>' + word + '</strong>' + percent);
+            studentWordsKeys.push(HtmlUtils.interpolateHtml(
+                gettext('{startTag}{word}{endTag}{percent}'),
+                {
```

Comment:
Remove the gettext usage here. There is nothing to translate-- these are all placeholders, so the translators will be very confused. Plus, we want the words to appear in whatever language they were entered in.